tests/BytesTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace IronCore;
+
+use PHPUnit\Framework\TestCase;
+
+final class BytesTest extends TestCase
+{
+    public function testBytesToHex(): void
+    {
+        $bytes = new Bytes("cow");
+        $hex = $bytes->getHexString();
+        $expected = "636f77";
+        $this->assertEquals($hex, $expected);
+
+        // Should get same result from toString implementation
+        $toString = strval($bytes);
```

Comment:
If`getHexString()` is interchangeable with `strval`, could it be removed?

Reply:
`strval` comes from our `__toString` implementation, which is a standard to implement so things like string concatenation work without unintended results. `getHexString()` was made so that there was an explicit function similar to `getByteString()` that was guaranteed to be all printable characters. So they are the same (and one just calls the other), but I think there's benefit to having both.